discord/package.json

```diff
@@ -50,6 +50,7 @@
     "@trshcmpctr/eslint-config": "workspace:*",
     "@trshcmpctr/markdownlint-config": "workspace:*",
     "@types/express": "~4.17.13",
+    "@typescript-eslint/parser": "~7.16.0",
```

Comment:
this is a peer dep of `@trshcmpctr/eslint-config-typescript` (because the client should be able to control the parser) but only detected as a missing peer since `pnpm@~9.1`